pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -222,6 +222,8 @@ func (pl *PodTopologySpread) calPreFilterState(pod *v1.Pod) (*preFilterState, er
 		TpKeyToCriticalPaths: make(map[string]*criticalPaths, len(constraints)),
 		TpPairToMatchNum:     make(map[topologyPair]*int32, sizeHeuristic(len(allNodes), constraints)),
 	}
+
+	var availableNodes []*framework.NodeInfo
```

Comment:
availableNodes -> filteredNodes.

And add a comment above:

```
// Nodes that pass nodeAffinity check and carry required topology keys will be
// stored in `filteredNodes`, and be looped later to calculate preFilterState.
```

Reply:
please take a look.